### PR TITLE
Translate Tips & FAQ section to English

### PR DESCRIPTION
**Pull request by duckfullstop**
This PR adds rudimentary translations for the Guides / Tips & FAQ section into English.

### CODE REVIEW

File: docs/guides/charging.mdx [hunk 1]
```diff
@@ -4,90 +4,89 @@ sidebar_position: 3
...
+The Minimum State of Charge feature supplies full power to the Vehicle if its State of Charge is lower than a defined minimum, in order to ensure that some kind of range is always available to use.
 
-Es ist ein Feature, dass mit voller Leistung bis zum Minimum-Soc geladen wird, wenn der Fahrzeug-Soc kleiner als der definierte Minimum-Soc ist.
-
-Wird der Fahrzeug-Soc nicht erkannt (= 0) und ist ein Minimum-Soc > 0 definiert, startet die Ladung.
-
-Die Ladung ist in dem Fall allerdings auf die Lademenge beschränkt, die sich aus Minimun-Soc und Fahrzeugkapazität ergibt.
+If the vehicle's State of Charge cannot be determined for some reason, charging starts to ensure that range is always available.
 
 ### "Probleme" i.V.m. Gastfahrzeug
```
Comment (duckfullstop) [inline docs/guides/charging.mdx:84]: I'm unsure how to translate this section, could someone else tackle it?

**Comment by duckfullstop** [comment]
Broke my branch a bit by accidentally committing against the DE versions, pardon the dust - commit history should be fixed now

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+### A note on individual tastes
+
+::::caution
+evcc has a lot of settings, knobs, and dials. The default parameters are based on lots of experience from many users, and most of the time, the default should work well for you.
+
+**If you do start making changes, remember the golden rule:**
+
+:::danger
+**Since parameters sometimes influence each other, it is STRONGLY encouraged to only make changes to one setting at at time, and test the effects of each change before making further modifications.**
+:::
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:18]: I wont do the nested blocks here. I think CAUTION+DANGER is a little much :D

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+
+**⛑️ First Aid**
+
+We've added a useful helper utility to evcc to help with the above. By entering `evcc discuss` in your terminal, our helper wizard will help you create a new discussion topic in our "First Aid" section.
+This discussion will automatically include any relevant error messages, and the anonymized configuration file - you only need to supply a meaningful title and a description of your problem.
+
+This First Aid utility can only be use for the initial creation of a discussion thread. If you need to supply further information to a topic already in progress, you'll need to do this manually.
+
+### How do I create a log fire for error analysis?
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:58]:
```suggestion
### How do I create a log file for error analysis?
```
🪵🔥😄

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx (hunk 2 above)
Comment (duckfullstop) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:18]: Think I agree here, but I wanna check on a laptop to see what looks best, gimme a bit 👀

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+`egrep -v '^#|^\s+#' evcc.yaml |grep . |sed -E 's/(password|user|token|url|host|broker).*/\1: *****/'`
+
+</details>
+
+  - Alternatively, you can use `evcc dump --cfg >~/config.txt` to create an anonymized configuration file at `~/config.txt`.
+
+
+**⛑️ First Aid**
+
+We've added a useful helper utility to evcc to help with the above. By entering `evcc discuss` in your terminal, our helper wizard will help you create a new discussion topic in our "First Aid" section.
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:53]: The topic in GitHub discussions is not localized and GH gives us no option to add localization. So we should keep Erste Hilfe" when describing the navigation. At least for now. Maybe add a notice that it's totally fine to also post in English.

Renaming the GH topic to English is something we should do eventually.

```suggestion
We've added a useful helper utility to evcc to help with the above. By entering `evcc discuss` in your terminal, our helper wizard will help you create a new discussion topic in our "Erste Hilfe" section.
```

Comment (duckfullstop) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:53]: Concur, should we make this read `"Erste Hilfe" (first aid)` to make it nice and clear what the section means?

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+When evcc is started directly from the console, log messages are printed directly to the screen (`STDOUT`).
+
+You can test out different parts of your installation using `evcc charger`, `evcc meter`, and `evcc vehicle`. Note that the log level defined in your `evcc.yaml` is used with these commands, but you can override it if necessary:
+
+`Example: evcc -l debug meter`
+
+If evcc is started as a Linux system service (as per [Linux Autostart](/docs/installation/manual#autostart)), you should be able to find your logs using the following commands:
+
+- Follow the log in real time
+  `sudo journalctl -fau evcc`
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:71]: Line wraps don't work. Not quite sure why.

Bildschirmfoto 2023-08-22 um 15 34 56

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+You can also activate this using the [`telemetry: true`](/docs/reference/configuration/telemetry/) key in the configuration, but this is a deprecated method and may be removed at a later date.
+
+**A 💚 Sponsor Token is currently required to participate in Community Data**. This helps ensure that our data quality stays high, and poor / fake data stays out.
+
+### What data is currently being collected?
+
+We currently collect the following:
+
+- current charging power
+- current proportion of charging power supplied by Solar
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:144]:
```suggestion
- current proportion of charging power supplied by solar
```

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+- total charged energy
+- total proportion of energy supplied by Solar
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:146]:
```suggestion
- total proportion of energy supplied by solar
```

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+
+If you charged 100% with your own solar energy, the displayed _effective energy price_ would be the cost of not exporting to the grid, i.e the feed-in tariff (€0.08/kWh).
+If you charge with 50% solar energy and 50% grid power, the _effective energy price_ adapts accordingly (e.g €0.187/kWh).
+
+If you don't receive a feed-in tariff for exporting power to the grid, you can set the feed-in price to 0 - the solar energy is then treated as being free of charge.
+If you have to pay tax on self-used energy, you can set the rate on your feed-in rate to get more realistic numbers.
+
+**Calculation of the solar energy share**
+
+If you get energy from more than one source at the same time (e.g 50% grid, 50% Solar), the algorithm will distribute the sources evenly across all configured Chargers.
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:211]:
```suggestion
If you get energy from more than one source at the same time (e.g 50% grid, 50% solar), the algorithm will distribute the sources evenly across all configured Chargers.
```

File: i18n/en/docusaurus-plugin-content-docs/current/guides/charging.mdx [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+
+Unfortunately, vehicles cannot be charged with any amount of power desired - there are minimum and maximum rates. These are defined by various standards:
+
+[IEC 61851](https://en.wikipedia.org/wiki/IEC_61851) (vehicle charging systems) defines a minimum charge speed of 6 Amps per phase,
+and depending on the configuration in `evcc.yaml`, a maximum of 16 or 32 amps.
+
+That means:
+
+- **Single-Phase Charging** - 1.4kW minimum, up to 3.6kW (16A) or 7.2kW (32A)
+- **Twin-Phase Charging** - 2.8kW minimum, up to 7.2kW (16A) or 14.4kW (32A)
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/charging.mdx:28]:
```suggestion
- **Two-Phase Charging** - 2.8kW minimum, up to 7.2kW (16A) or 14.4kW (32A)
```

See example here: https://go-e.com/en/magazine/single-phase-three-phase-ev-charging

File: i18n/en/docusaurus-plugin-content-docs/current/guides/faq.mdx [hunk 10]
```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 7
+---
+
+# Miscellaneous
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/faq.mdx:5]: This really feels like `Miscellaneous` that doesn't fit anywhere else. 

Maybe we should remove it of find a proper location later. But this is a content discussion and not something for this translation PR.

\\cc @VolkerK62

File: i18n/en/docusaurus-plugin-content-docs/current/guides/meters.mdx [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+# Meters / Site
+
+### Are my metering devices supported?
+
+Hopefully (and relatively likely!). Have a look in [Devices - Meters](/docs/devices/meters) to see if your device is listed.
+If it isn't, try a configuration using a generic interface, or a similar device from the same brand.
+
+Many Inverters are Sunspec-compatible (even if it sometimes needs to be enabled), so it's worth trying to find out whether yours supports either this or another generic communication standard.
+
+### I have a Solar installation, but I don't have a (readable) Grid connection meter - can I still use evcc?
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/meters.mdx:16]:
```suggestion
### I have a solar installation, but I don't have a (readable) grid connection meter - can I still use evcc?
```

File: i18n/en/docusaurus-plugin-content-docs/current/guides/vehicles.mdx [hunk 12]
```diff
@@ -0,0 +1,198 @@
...
+9. Set the corresponding identifiers in your evcc.yaml:
+  ```
+  - name: merc
+    type: mercedes
+    title: MyReallyCoolMerc
+    capacity: 80 # battery capacity kWh
+    clientId: xxx
+    clientSecret: xxx
+    vin: xxxxx # vin number
+  ```
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/vehicles.mdx:198]: Formatting issue

Bildschirmfoto 2023-08-22 um 15 59 45

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx (hunk 2 above)
Comment (duckfullstop) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:19]: Resolved in [af64646](https://github.com/evcc-io/docs/pull/426/commits/af646464ccfae587ce7dffba35428f2c5138968e)

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx (hunk 5 above)
Comment (duckfullstop) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx]: Probably worth investigating why this is happening with the element, might need to set a wrapping CSS rule on inline code elements

For now, resolved in [988f08e](https://github.com/evcc-io/docs/pull/426/commits/988f08e72acab3818a800f013e87c923614d40d5)

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx [hunk 13]
```diff
@@ -0,0 +1,215 @@
...
+You might find more useful commands here: [wiki.archlinux.org/title/Systemd/Journal](https://wiki.archlinux.org/title/Systemd/Journal#Filtering_output)
+
+If you're using Docker, you should be able to use `docker logs`. See the [Docker documentation](https://docs.docker.com/config/containers/logging/) for more details.
+
+### More thoughts on device detection
+
+`evcc detect` is a special command that attempts to find compatible hardware on your network.
+In particular, it can sometimes help find "new" Sunspec-compatible modbus devices - however, it is more of a developer / support tool for diagnostic purposes, and can't provide detailed results.
+
+## Common Errors
```
[the remark this answers is not in the document]
Comment (naltatis) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx:88]: Sorry. I wasn't clear in expressing what I thought :) Yes, we keep this change and update DE afterwards.

File: i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx (hunk 4 above)
Comment (naltatis) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx]: Good point. I changed it.

**Comment by naltatis** [comment]
Looks good to me. I'll merge this. Thanks a lot 💚.

Quick note: I noticed that my editor reformatted some of your files. He have a prettier configuration for this project that formats all source files. You can run `npm run format` manually or use a format-on-save plugin for your editor.

**Pull request merged**